src/kubernetes/api/util.clj

```diff
@@ -27,21 +29,34 @@
        (if (empty? query) "" "?")
        (query-str query)))
 
+(defn- token [username password]
+  (when (and username password)
+    (str username ":" password)))
```

Comment:
Are you sure about this format? The [documentation says](http://www.http-kit.org/client.html) it requires `:basic-auth` to be a vector, like `{:basic-auth ["user" "pass"]}` 🤔

Reply:
It works. :man_shrugging:

Reply:
:ata: